test/e2e-common.sh

```diff
@@ -210,6 +216,13 @@ function install_knative_serving_standard() {
   echo ">> Bringing up Serving"
   kubectl apply -f "${INSTALL_RELEASE_YAML}" || return 1
 
+  if (( RECONCILE_GATEWAY )); then
+    echo ">>Turning on reconcileExternalGateway."
+    kubectl get cm config-istio -n knative-serving -o yaml | \
+      sed 's/  reconcileExternalGateway: \"false\"/reconcileExternalGateway: \"true\"/g' |\
```

Comment:
I assume you DO want to remove the two spaces before the key.

```suggestion
      sed 's/  reconcileExternalGateway: "false"/reconcileExternalGateway: "true"/g' | \
```

Reply:
yes, I need to remove the first two leading spaces.